.changelogs/issue_2063-1.yml

```
@@ -0,0 +1,5 @@
+significance: patch
```

A new filter is being added.
```suggestion
significance: minor
```

---

I think you're right, thanks.